From: D Haley <mycae@gmx.com>
Date: Tue, 26 Mar 2019 14:59:11 +0000
Subject: Build shared qhullcpp

---
 CMakeLists.txt | 18 ++++++++++++++----
 1 file changed, 14 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index bf7a259..eed1927 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -341,6 +341,7 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR}/src)
 
 if(CMAKE_BUILD_TYPE MATCHES "[dD]ebug")
     set(qhull_CPP qhullcpp_d)
+    set(qhull_CPPSTATIC qhullcpp_static_d)
     set(qhull_SHARED qhull_d) 
     set(qhull_SHAREDP qhull_pd)
     set(qhull_SHAREDR qhull_rd)
@@ -348,6 +349,7 @@ if(CMAKE_BUILD_TYPE MATCHES "[dD]ebug")
     set(qhull_STATICR qhullstatic_rd)
 else()
     set(qhull_CPP qhullcpp)
+    set(qhull_CPPSTATIC qhullcpp_static)
     set(qhull_SHARED libqhull)  # Temporarily avoid name conflict with qhull executable
     set(qhull_SHAREDP qhull_p)
     set(qhull_SHAREDR qhull_r)
@@ -464,14 +466,22 @@ if(UNIX)
 endif(UNIX)
 
 # ---------------------------------------
-# Define C++ static library qhullcpp
-#     Do not create libqhullcpp as a shared library.  Qhull C++ classes may change layout and size. 
+# Define C++ shared library qhullcpp
 # ---------------------------------------
 
-add_library(${qhull_CPP} STATIC ${libqhullcpp_SOURCES})
+add_library(${qhull_CPP} SHARED ${libqhullcpp_SOURCES})
 set_target_properties(${qhull_CPP} PROPERTIES
     VERSION ${qhull_VERSION}
-    POSITION_INDEPENDENT_CODE "TRUE")
+    SOVERSION ${qhull_SOVERSION}
+)
+target_link_libraries(${qhull_CPP} ${qhull_SHAREDR})
+
+add_library(${qhull_CPPSTATIC} STATIC ${libqhullcpp_SOURCES})
+set_target_properties(${qhull_CPPSTATIC} PROPERTIES
+    VERSION ${qhull_VERSION}
+    OUTPUT_NAME "${qhull_CPP}"
+)
+target_link_libraries(${qhull_CPPSTATIC} ${qhull_STATICR})
 
 # ---------------------------------------
 # Define qhull executables linked to qhullstatic library
