From: =?utf-8?q?Timo_R=C3=B6hling?= <timo.roehling@fkie.fraunhofer.de>
Date: Mon, 6 Apr 2020 18:58:53 +0200
Subject: Fix CMake target export

---
 CMakeLists.txt | 18 +++++++++++++-----
 1 file changed, 13 insertions(+), 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index eed1927..ff12517 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -358,10 +358,13 @@ else()
 endif()
 
 set(
-    qhull_TARGETS_INSTALL
+    qhull_EXPORTED_TARGETS_INSTALL
         ${qhull_CPP} ${qhull_STATIC} ${qhull_STATICR} ${qhull_SHAREDR}
+	${qhull_SHARED} ${qhull_CPPSTATIC})
+set(
+    qhull_OTHER_TARGETS_INSTALL
         qhull rbox qconvex qdelaunay qvoronoi qhalf
-        ${qhull_SHARED} ${qhull_SHAREDP}  # Deprecated, use qhull_r instead
+	${qhull_SHAREDP}  # Deprecated, use qhull_r instead
 )
 set(
     qhull_TARGETS_TEST   # Unused
@@ -632,7 +635,12 @@ add_test(NAME user_eg3
 # Define install
 # ---------------------------------------
 
-install(TARGETS ${qhull_TARGETS_INSTALL} EXPORT QhullTargets
+install(TARGETS ${qhull_EXPORTED_TARGETS_INSTALL} EXPORT QhullTargets
+        RUNTIME DESTINATION ${BIN_INSTALL_DIR}
+        LIBRARY DESTINATION ${LIB_INSTALL_DIR}
+        ARCHIVE DESTINATION ${LIB_INSTALL_DIR}
+        INCLUDES DESTINATION include)
+install(TARGETS ${qhull_OTHER_TARGETS_INSTALL}
         RUNTIME DESTINATION ${BIN_INSTALL_DIR}
         LIBRARY DESTINATION ${LIB_INSTALL_DIR}
         ARCHIVE DESTINATION ${LIB_INSTALL_DIR}
@@ -656,7 +664,7 @@ configure_file(${PROJECT_SOURCE_DIR}/build/config.cmake.in
   @ONLY
 )
 
-set(ConfigPackageLocation lib/cmake/Qhull)
+set(ConfigPackageLocation ${LIB_INSTALL_DIR}/cmake/Qhull)
 install(EXPORT QhullTargets
   FILE
     QhullTargets.cmake
@@ -675,7 +683,7 @@ install(
     Devel
 )
 
-set(PkgConfigLocation lib/pkgconfig)
+set(PkgConfigLocation ${LIB_INSTALL_DIR}/pkgconfig)
 foreach(pkgconfig IN ITEMS "${qhull_SHARED};Qhull shared library"
                            "${qhull_SHAREDR};Qhull reentrant shared library"
                            "${qhull_STATIC};Qhull static library"
