From: D Haley <mycae@gmx.com>
Date: Tue, 26 Mar 2019 14:59:11 +0000
Subject: qhullcpp-shared

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 59dff41..fc4701f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -459,7 +459,7 @@ endif(UNIX)
 #     Do not create libqhullcpp as a shared library.  Qhull C++ classes may change layout and size. 
 # ---------------------------------------
 
-add_library(${qhull_CPP} STATIC ${libqhullcpp_SOURCES})
+add_library(${qhull_CPP} SHARED ${libqhullcpp_SOURCES})
 set_target_properties(${qhull_CPP} PROPERTIES
     VERSION ${qhull_VERSION})
 
