From: =?utf-8?q?Timo_R=C3=B6hling?= <timo.roehling@fkie.fraunhofer.de>
Date: Sat, 11 Apr 2020 12:32:14 +0200
Subject: Do not set RPATH

---
 CMakeLists.txt | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ff12517..710312a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -387,11 +387,6 @@ if(UNIX)
     if(APPLE)
         set_target_properties(${qhull_SHAREDR} PROPERTIES 
             INSTALL_NAME_DIR "${LIB_INSTALL_DIR}")
-    else()
-        set_target_properties(${qhull_SHAREDR} PROPERTIES 
-            INSTALL_RPATH "${LIB_INSTALL_DIR}"
-            INSTALL_RPATH_USE_LINK_PATH TRUE
-            BUILD_WITH_INSTALL_RPATH FALSE)
     endif()
 endif(UNIX)
 
@@ -418,11 +413,6 @@ if(UNIX)
     if(APPLE)
         set_target_properties(${qhull_SHARED} PROPERTIES 
             INSTALL_NAME_DIR "${LIB_INSTALL_DIR}")
-    else()
-        set_target_properties(${qhull_SHARED} PROPERTIES 
-            INSTALL_RPATH "${LIB_INSTALL_DIR}"
-            INSTALL_RPATH_USE_LINK_PATH TRUE
-            BUILD_WITH_INSTALL_RPATH FALSE)
     endif()
 endif(UNIX)
 
@@ -443,11 +433,6 @@ if(UNIX)
     if(APPLE)
         set_target_properties(${qhull_SHAREDP} PROPERTIES 
             INSTALL_NAME_DIR "${LIB_INSTALL_DIR}")
-    else()
-        set_target_properties(${qhull_SHAREDP} PROPERTIES 
-            INSTALL_RPATH "${LIB_INSTALL_DIR}"
-            INSTALL_RPATH_USE_LINK_PATH TRUE
-            BUILD_WITH_INSTALL_RPATH FALSE)
     endif()
 endif(UNIX)
 
