--- qhull-orig/src/io.c	2004-01-31 12:00:15.000000000 +0100
+++ qhull/src/io.c	2004-02-06 09:54:18.000000000 +0100
@@ -4017,11 +4017,8 @@
     qh rbox_command[strlen(qh rbox_command)-1]= '\0';
     if (!strcmp (qh rbox_command, "./rbox D4")) 
       fprintf (qh ferr, "\n\
-This is the qhull test case.  If any errors or core dumps occur,\n\
-recompile qhull with 'make new'.  If errors still occur, there is\n\
-an incompatibility.  You should try a different compiler.  You can also\n\
-change the choices in user.h.  If you discover the source of the problem,\n\
-please send mail to qhull_bug@qhull.org.\n\
+This is the qhull test case.  If any errors or core dumps occur,\n\
+fill a bug report against the libqhull<soversion> Debian package.\n\
 \n\
 Type 'qhull' for a short list of options.\n");
   }
